### ISSUE TITLE
New Fibaro Wall Plugs not work 

### ISSUE THREAD

**Issue by selzilla** [athombv/homey#1656]
Problem with the Fibaro Wall Plugs z wave plus

Newly harvested equipment is only available for a short time via Homey.
They are displayed permanently, but they no longer react to flows and manual switching.
Before the current update plugs work perfectly.
Learning new did not succeed. Only a multiple start of Homey makes the plugs switchable again. However, this does not last long before the plugs can not be reached again.

**What is your Homey version (Settings → System)?**

1.3.2

**Comment by selzilla** [comment]
I believe I have identified the cause. I have observed several times that the memory was completely exhausted. As soon as this happens, there is probably a buffer overflow. I have now deleted all unnecessary apps to get some memory free. On my sorage is place without end. Is there any way to outsource memory to the storage? If I think of what Homey would like to integrate in the future, we will get a real problem with the working memory. Since I have made some memory free, my Wallplugs can be switched again. It bothers me very much, however, that I can not install new apps without having to worry about a memory overflow again. I will observe, o my presumption is confirmed for a longer term.
